rest-service/manager_rest/cluster_status_manager.py

```diff
@@ -54,36 +54,45 @@
     'blackbox_exporter': CloudifyService(
         name='Blackbox Exporter',
         description='Prometheus blackbox exporter (HTTP/HTTPS/TCP pings)'),
+    'cloudify-amqp-postgres': CloudifyService(
+        name='AMQP-Postgres',
+        description='Cloudify AMQP PostgreSQL Broker service'),
+    'cloudify-composer': CloudifyService(
+        name='Cloudify Composer',
+        description='Cloudify Composer service'),
+    'cloudify-mgmtworker': CloudifyService(
+        name='Management Worker',
+        description='Cloudify Management Worker service'),
+    'cloudify-rabbitmq': CloudifyService(
+        name='RabbitMQ Broker',
+        description='RabbitMQ Broker service'),
+    'cloudify-restservice': CloudifyService(
+        name='Manager REST',
+        description='Cloudify REST service'),
+    'cloudify-stage': CloudifyService(
+        name='Cloudify Console',
+        description='Cloudify Console service'),
+    'cloudify-syncthing': CloudifyService(
+        name='Syncthing',
+        description='Syncthing service'),
     'haproxy': CloudifyService(
         name='HA Proxy',
-        description='HAProxy Load Balancer'),
+        description='HAProxy Load Balancer service'),
```

Comment:
Hmm... do we need all of the services to have 'service' at the end of their description? I'd probably have gone with removing it from the others rather than adding it to this (and maybe making the descriptions more meaningful such as syncthing's being 'File synchronisation between manager nodes' and console being 'Cloudify Web UI' or something?